crates/net/rpc/src/trace.rs

```diff
@@ -0,0 +1,95 @@
+use async_trait::async_trait;
+use jsonrpsee::core::RpcResult as Result;
+use reth_network::NetworkHandle;
+use reth_primitives::{rpc::BlockId, Bytes, H256};
+use reth_rpc_api::TraceApiServer;
+use reth_rpc_types::{
+    trace::{filter::TraceFilter, parity::*},
+    CallRequest, Index,
+};
+use std::collections::HashSet;
+
+/// `trace` API implementation.
+///
+/// This type provides the functionality for handling `trace` related requests.
+pub struct TraceApi {
+    network: NetworkHandle,
```

Comment:
we'll phase out `NetworkHandle` entirely, see #979 

so let's rm that here, need to figure out first what we actually need.

Reply:
i see, will update